src/components/AuthErrorsContainer/AuthErrorsContainer.js

```diff
@@ -21,7 +21,12 @@ export default class AuthErrorsContainer extends Component {
     const values = parameters.reduce((res, { key, value }) => ({ ...res, [key]: value }), {});
 
     return (
-      <li key={`${code}-${type}`}>
+      <li
+        key={`${code}-${type}`}
+        aria-invalid
```

Comment:
To my knowledge, `aria-invalid` is primarily used for form elements and should not have any effect in this case. But you might have found otherwise so I'm curious to hear if you found this attribute to make a difference here?

Reply:
No, you are right, thats my mistake, I will remove `aria-invalid` attribute